### DISCUSSION TITLE
PnP - Search Results will exclude the document that have been copied

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#2800] [Q&A]
I have a document library which contain around 7500 documents:-

![image](https://user-images.githubusercontent.com/79726511/220986304-e0764ed7-a635-422c-872f-c5b24b5e264c.png)

and i defined the following PnP - Search Result web part >> 

![image](https://user-images.githubusercontent.com/79726511/220986410-075928a5-c9eb-4094-8a50-aa44bb850317.png)

but the content search is only showing 9 documents >> 

![image](https://user-images.githubusercontent.com/79726511/220987253-b22c8639-7ca9-43b4-b8ca-24e085281b90.png)

where those 9 documents were manually uploaded, while the remaining documents were copied using Power Automate workflow or manually  from the 9 documents.. so is there a reason why those documents are been excluded ? and can i include them? as those documents are showing inside the SharePoint built in search box, which means those documents were indexed...

**Comment by wobba** [comment]
Wait up to 72h and recheck if items are indexed.

**Reply by mvcsharepointdev** [reply]
@wobba those items were added 1 week ago, so for sure there are indexed + they are showing inside sharepoint built-in search

**Reply by wobba** [reply]
If you see them in the OOB search experience (site or orgwide search, not library search) , then likely something with your configuration.
